Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2005-08-27
Initial Package Version: 4.x
Origin: Ken Moffat and Jim Gifford
        Rediffed against 4.1.0 by Chris Staub
Upstream Status: LFS Specific
Description: Make LIB64 builds in /lib instead of /lib64 

diff -Naur gcc-4.1.0.orig/gcc/config/i386/linux64.h gcc-4.1.0/gcc/config/i386/linux64.h
--- gcc-4.1.0.orig/gcc/config/i386/linux64.h	2005-08-10 13:53:01.000000000 -0400
+++ gcc-4.1.0/gcc/config/i386/linux64.h	2006-03-01 01:13:54.000000000 -0500
@@ -60,8 +60,8 @@
   %{!shared: \
     %{!static: \
       %{rdynamic:-export-dynamic} \
-      %{m32:%{!dynamic-linker:-dynamic-linker /lib/ld-linux.so.2}} \
-      %{!m32:%{!dynamic-linker:-dynamic-linker /lib64/ld-linux-x86-64.so.2}}} \
+      %{m32:%{!dynamic-linker:-dynamic-linker /lib32/ld-linux.so.2}} \
+      %{!m32:%{!dynamic-linker:-dynamic-linker /lib/ld-linux-x86-64.so.2}}} \
     %{static:-static}}"
 
 /* Similar to standard Linux, but adding -ffast-math support.  */
diff -Naur gcc-4.1.0.orig/gcc/config/i386/t-linux64 gcc-4.1.0/gcc/config/i386/t-linux64
--- gcc-4.1.0.orig/gcc/config/i386/t-linux64	2005-08-10 13:53:01.000000000 -0400
+++ gcc-4.1.0/gcc/config/i386/t-linux64	2006-03-01 01:13:54.000000000 -0500
@@ -6,7 +6,7 @@
 
 MULTILIB_OPTIONS = m64/m32
 MULTILIB_DIRNAMES = 64 32 
-MULTILIB_OSDIRNAMES = ../lib64 ../lib
+MULTILIB_OSDIRNAMES = ../lib ../lib32
 
 LIBGCC = stmp-multilib
 INSTALL_LIBGCC = install-multilib
